guides/hack/08-generic-types-and-functions/21-reified-generics-migration.md

```diff
@@ -0,0 +1,66 @@
+# Migration Features for Reified Generics
+
+In order to make migrating to reified generics easier, we have added the following migration features:
+
+* The `<<__Soft>>` annotation on a type parameter implies the intent to switch it to reified state. If generics at call sites are not provided the type checker will raise an error, and the runtime will raise a warning so you can capture these locations through logging. The generic cannot be used in the body of the function.
+* The `<<__Warn>>` annotation on a type parameter implies that if the reified generic is incorrect at parameter or return type hints locations, the runtime will raise a warning instead of type hint violation.
+* The `getType<reify T>()` function, given a reified type, returns the type structure representing the type.
```

Comment:
Is `getType()` the fully qualified name? Is it an instance method on some class or a function?